spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/catalog/BaseCatalog.java

```diff
@@ -162,10 +164,10 @@ public Identifier[] listTables(String[] namespace) throws NoSuchNamespaceExcepti
     }
   }
 
+  @SneakyThrows
```

Comment:
what's the benefit of using `@SneakyThrows`?

Reply:
> SneakyThrows

`SneakyThrows` can automatically generate a try catch in `.class` file when it is troublesome to handle exceptions manually.

Reply:
I prefer to remove `@SneakyThrows`, because it's more clear to handle the exception explicitly

Reply:
> I prefer to remove `@SneakyThrows`, because it's more clear to handle the exception explicitly

ok

Reply:
updated.